org.eclipse.winery.repository.ui/src/app/wineryRepository.component.ts

```diff
@@ -26,7 +26,7 @@ import { backendBaseURL } from './configuration';
 export class WineryRepositoryComponent implements OnInit {
     // region variables
     name = 'Winery Repository';
-    isBackendAvailable = false;
```

Comment:
Are you sure that this should be changed? Doesn't that lead to errors during loading if the backend is not available?

Reply:
i changed the default value to true and only if the backend is not available then we set it to false. Otherwise we had some strange behaviour that the reload animation doens't show up always

Reply:
I think, Winery continues loading when the backend is not available with this if.

I would make two different PRs, as this fixes are two completely different issues.

Reply:
Currently "Backend is not available" is a boolean state leading to "Backend is not available" being always displayed.

It should be a tri-state:

- undefined
- true
- false

In the undefined case: "Loading..." should be displayed. The other behavior as is.